src/components/form/day-picker/index.tsx

```diff
@@ -184,11 +193,11 @@ export const DayPicker = ({
 
   const [range, setRange] = useState<DateRange | undefined>(defaultRangeSelected);
 
-  const defaultDate = value || defaultDay || new Date();
+  const defaultDate = value || defaultDay || null;
 
   const [isCalendarOpen, setIsCalendarOpen] = useState(false);
-  const [date, setDate] = useState<Date>(defaultDate);
-  const [dateTime, setDateTime] = useState(transformTime(new Date(date)));
+  const [date, setDate] = useState<Date | null>(defaultDate);
```

Comment:
do we really need `date` and `dateTime`?

I feel like it's a bit redundant, single date transformation would be better.

Reply:
@lazzystep 
The component is deprecated. Hopefully, it's the last change to it. The logic inside is hard to customize and maintain, so the suggestion is to leave it like it is